### ISSUE TITLE
View Submission Interactivity handler response is incorrect

### ISSUE THREAD

**Issue by robcoward** [hanchchch/nestjs-slack-listener#253]
As per https://api.slack.com/surfaces/modals#close_current_view - when processing a `view_submission` payload sent to the slack interactivity URL, in order to close a view, the `200` response either needs to be accompanied by an empty payload, or by returning
```
{
  "response_action": "clear"
}
```
The current handling of interaction payloads results in a payload that is a list of all the defined handler responses, similar to below:
```
[
  null,
  null,
  { response_action: 'clear' },
  null,
  null
]
```
Slack does not recognise this as a valid response and shows as an error in the modal when trying to submit the form.

This problem arises in the `handleInteractivity(payload: IncomingSlackInteractivity)` method in `slack-handler-service.ts` where `this._interactivityHandlers.map()` is returning a list of the responses from each `this.handleSingleInteractivity(payload, handlerConfig)` call.

**Comment by robcoward** [comment]
Fixed by https://github.com/hanchchch/nestjs-slack-listener/pull/252

**Comment by hanchchch** [comment]
Resolved in version 1.1.3. Thanks for fixing it!
